tests/js/common/shell/shell-pregel-components.js

```diff
@@ -418,14 +417,20 @@ const hitsTestSuite = pregelTestHelpers.makeHITSTestSuite(false, "", 4);
 
 // const effectiveClosenessTestSuite = pregelTestEffectiveClosenessHelpers.makeEffectiveClosenessTestSuite(false, ", 4");
 
-jsunity.run(componentsTestSuite);
-jsunity.run(wccRegressionTestSuite);
-jsunity.run(wccTestSuite);
-jsunity.run(sccTestSuite);
-jsunity.run(labelPropagationTestSuite);
-jsunity.run(pagerankTestSuite);
-jsunity.run(seededPagerankTestSuite);
-jsunity.run(ssspTestSuite);
-jsunity.run(hitsTestSuite);
+const readWriteTestSuite = pregelTestReadWriteHelpers.makeReadWriteTestSuite(false, "", 4);
+
+// jsunity.run(componentsTestSuite);
+// jsunity.run(wccRegressionTestSuite);
+// jsunity.run(wccTestSuite);
+// jsunity.run(sccTestSuite);
+// jsunity.run(labelPropagationTestSuite);
+// jsunity.run(pagerankTestSuite);
+// jsunity.run(seededPagerankTestSuite);
+// jsunity.run(ssspTestSuite);
+// jsunity.run(hitsTestSuite);
 // jsunity.run(effectiveClosenessTestSuite);
```

Comment:
Why is all this commented out?

Reply:
Oh, thank you! It was for debugging and I forgot to un-comment it. Reverted now.